packages/themes/src/reset.css

```diff
@@ -75,4 +80,15 @@
     outline: 1px solid var(--scalar-color-accent);
     outline-offset: 1px;
   }
+
+  /** Set the default cursor for buttons. */
+  button,
```

Comment:
seems like the button in api client are tight off, should we inherit their display?

Reply:
I'm not sure what you mean by tight off?

Reply:
sorry typo 😂 i meant within api client the sidebar icon button + history button are bigger than usual

Reply:
Ahhh okay, I'll have a look

Reply:
Fixed!